Source/ProgressionSystemRuntime/Public/Widgets/PSOverlayWidget.h

```diff
@@ -13,21 +13,58 @@ UCLASS()
 class PROGRESSIONSYSTEMRUNTIME_API UPSOverlayWidget : public UUserWidget
 {
 	GENERATED_BODY()
+
 public:
-	
+	/**
+	* Sets the visibility of the overlay elements and playing fade animation if needed 
+	* @param VisibilitySlate The visibility state (e.g., Visible, Collapsed) to apply to the overlay and icon.
+	* @param bShouldPlayFadeAnimation Defines if the fade animation should be played after or before widget is visible
+	*/
+	UFUNCTION(BlueprintCallable, Category= "C++", meta = (BlueprintProtected))
```

Comment:
Please remove `BlueprintProtected`, it's under `public`

Reply:
remvoed with https://github.com/h4rdmol/ProgressionSystem/pull/20/commits/5a093a105852f504a06cc46d8b2d67fe65cb6918